<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoiceChatWin</class>
 <widget class="QWidget" name="VoiceChatWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VoiceChatWin</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(250, 250, 250);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>270</width>
     <height>220</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(30, 180, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>105</x>
      <y>50</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/05-01.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>120</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>等待对方接收邀请...</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Mini">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>35</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/05-06.png</normaloff>images/05-06.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Close">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>0</y>
      <width>35</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/05-07.png</normaloff>images/05-07.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="Micro">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>35</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/05-02.png</normaloff>images/05-02.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Volume">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>230</y>
     <width>35</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/05-03.png</normaloff>images/05-03.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Video">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>45</width>
     <height>40</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/05-04.png</normaloff>images/05-04.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="Signal">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>235</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/05-05.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="Hang_up">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>232</y>
     <width>65</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
border:0px;
color: rgb(255, 255, 255);
background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>挂断</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Answer">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>232</y>
     <width>65</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
border:0px;
color: rgb(255, 255, 255);
background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>接听</string>
   </property>
  </widget>
  <widget class="QSlider" name="Micro_volume_Slider">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>70</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(194, 194, 194);</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QSlider" name="Speaker_volume_Slider">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(194, 194, 194);</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>2</number>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Mini</sender>
   <signal>clicked()</signal>
   <receiver>VoiceChatWin</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
